scrapy/czlscrape/spiders/cultura.py

```diff
@@ -0,0 +1,217 @@
+# coding=utf-8
+
+import datetime
+import hashlib
+import re
+
+from scrapy import Spider, Request
+from scrapy.selector import Selector
+
+from ..items import Publication
+
+CSS_ARTICLE_LIST_TITLES = '#recomended-articles .recommended-title'
+CSS_ARTICLE_TITLE = '#block-continutprincipalpagina > div > article > div > div.post-content > h1 > span'
+CSS_ARTICLE_TEXT = '#block-continutprincipalpagina > div > article > div > div.post-content > div.node__content.clearfix > div'
+CSS_ARTICLE_PUBLISH_DATE = '#block-continutprincipalpagina > div > article > div > div.post-content > div.post-meta > span.post-created'
+CSS_ARTICLE_LINKS = 'ul > li > a, a'
+
+INDEX_URL = 'http://www.cultura.ro/proiecte-acte-normative'
+
+PUBLISH_DATE_PATTERN = r'''
+^                                           # beginning of string
+(?P<day>\d{1,2})                            # day is one or two digits
+\s                                          # whitespace
+(?P<month>\w*)                              # name of the month in Romanian
+\s                                          # whitespace
+(?P<year>\d{4})                             # year is 4 digits
+$                                           # end of string
+'''
+
+FEEDBACK_DEADLINE_DATE_PATTERN = r'.*(?:pana la data de|pana la data) (?P<day>\d{1,2})\.(?P<month>\d{1,2})\.(?P<year>\d{4}).*'
+
+FEEDBACK_CONTACT_FAX_PATTERN = r'''
+.*                                          # can start anywhere
+fax                                         # the relevant prefix
+:?                                          # optional colon
+\s?                                         # optional whitespace
+(?:la numarul|la numar|la nr\.|la nr)?      # optional prefix
+\s?                                         # optional whitespace
+(\S*)                                       # fax number
+.*                                          # anywhere in the string
+'''
+
+FEEDBACK_CONTACT_EMAIL_PATTERN = r'''
+.*                                          # can start anywhere
+e-mail:?                                    # the relevant prefix
+\s?                                         # optional whitespace
+(\S*)                                       # email address
+.*                                          # anywhere in the string
+'''
+
+FEEDBACK_CONTACT_RULES = [
+    ('fax', FEEDBACK_CONTACT_FAX_PATTERN),
+    ('email', FEEDBACK_CONTACT_EMAIL_PATTERN),
+]
+
+ROMANIAN_MONTHS = {
+    'ianuarie': 1,
+    'februarie': 2,
+    'martie': 3,
+    'aprilie': 4,
+    'mai': 5,
+    'iunie': 6,
+    'iulie': 7,
+    'august': 8,
+    'septembrie': 9,
+    'octombrie': 10,
+    'noiembrie': 11,
+    'decembrie': 12
```

Comment:
Lipsește virgulă la ultimul element. Nu țin morțiș la convenția asta, dar să fie consistent fișierul, one way or the other.